library/Graphite/Web/Controller/MonitoringAwareController.php

```diff
@@ -20,4 +24,14 @@ protected function applyMonitoringRestriction(DataView $dataView)
 
         return $dataView;
     }
+
+    protected function moduleInit()
+    {
+        if (Module::exists('icingadb') && IcingadbSupport::useIcingaDbAsBackend()) {
+            return;
+        }
+
+        $this->backend = Backend::createBackend($this->_getParam('backend'));
+        $this->view->url = Url::fromRequest();
```

Comment:
Because I use the same ListController for the URL, an error is thrown if no IdoBackend resource is defined (No Backend set). Therefore, I override the method so that the controller does not need the Ido backend resource if icingadb is set.

Reply:
Aaaah.. and why don't you call the parent method then?

Reply:
You've removed it entirely. It sets up the monitoring backend for no reason now. This happens in case of the redirects and for every single graph. Please add it again.